libsplinter/src/biome/user/store/mod.rs

```diff
@@ -56,7 +56,7 @@ impl From<UserModel> for SplinterUser {
 
 /// Defines methods for CRUD operations and fetching and listing users
 /// without defining a storage strategy
-pub trait UserStore<T> {
+pub trait UserStore<T>: Send + Sync + Clone {
```

Comment:
Using Clone this way, we can't create `Box<dyn UserStore>` (the compiler will complain).  We probably don't need this constraint, yet until we start using `UserStore` as a trait.  `NodeRegistry` is a good example of how we have to handle it.

Reply:
Added CloneBoxUserStore trait